utils/utils.ts

```diff
@@ -28,6 +28,8 @@ export function fromBNToUnitString(value : BN){
 
 export function getEnvironmentRequiredVars(){
     const palletAddress = process.env.TEST_PALLET_ADDRESS ? process.env.TEST_PALLET_ADDRESS : '';
+    const palletAddress = process.env.XYK_PALLET_ADDRESS ? process.env.XYK_PALLET_ADDRESS : '';
+    const palletAddress = process.env.TREASURY_PALLET_ADDRESS ? process.env.TREASURY_PALLET_ADDRESS : '';
```

Comment:
we need to differentiate between those 3 addresses, do you mind naming them properly and adding them to the return sentence? 
Afterwards, I can update the tests, I think I know which tests must read each of those.